addons/onion-skinning/addon.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "Onion skinning",
+  "description": "Shows transparent overlays of previous or next costumes when editing a costume. Controlled by buttons under the editor by the zoom buttons.",
+  "credits": [
+    {
+      "name": "GarboMuffin"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/projects/*"]
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "style.css",
+      "matches": ["https://scratch.mit.edu/projects/*"]
+    }
+  ],
+  "settings": [
+    {
+      "id": "default",
+      "name": "Enable by default",
+      "type": "boolean",
+      "default": false
+    },
+    {
+      "id": "previous",
+      "name": "Default previous costumes",
+      "type": "positive_integer",
+      "default": 1
+    },
+    {
+      "id": "next",
+      "name": "Default next costumes",
+      "type": "positive_integer",
+      "default": 0
+    },
+    {
+      "id": "opacity",
+      "name": "Opacity",
+      "type": "integer",
+      "min": 0,
+      "max": 100,
+      "default": 25
+    },
+    {
+      "id": "opacityStep",
+      "name": "Opacity Step",
+      "type": "integer",
+      "min": 0,
+      "max": 100,
+      "default": 10
+    }
+  ],
+  "tags": ["editor", "beta"],
```

Comment:
"Beta" can be removed (we'll have testers test the addon before the release) and this should be recommended since it's a very suggested Scratch feature.

Reply:
Yay!